_episodes/03-files.md

```diff
@@ -106,7 +107,7 @@ $ ls
 ## Moving and copying files
 
 To practice moving files, we will move `test.txt` to that directory with `mv` (move). `mv`'s syntax
-is relatively simple, and works for both files and directories `mv <file/directory> <path to new
+is relatively intuitive, and works for both files and directories `mv <file/directory> <path to new
```

Comment:
I'd drop formulations like `is relatively intuitive`, as it could give people who don't immediately grasp `mv` mechanics a weird learning experience.